turistické-cíle/draft/příklady/turistický_cíl.json

```diff
@@ -9,7 +9,10 @@
     "obrázek": "https://upload.wikimedia.org/wikipedia/commons/4/47/Mesto_Brno_-_hrad_Spilberk.jpg",
     "kouření_povoleno": true,
     "veřejná_přístupnost": true,
-    "kapacita": "500 lidí",
+    "kapacita": {
+        "hodnota": 500,
+        "jednotka": "c62"
```

Comment:
Tady to má být C62, ne c62

Reply:
@dvoriik to je asi na tebe toto. díky za vyřešení. J

Reply:
@jakubklimek . Opraveno, odkaz zde --> [https://github.com/jirkamarekbrno/otevrene-formalni-normy/commit/35d7a3c8d96adc6cc0b8900544f9aadf1dc7f13e](url)